### PR TITLE
Library app 

### PR DESCRIPTION
**Pull request by MarkoNS1990**
🟢 In this project, I built a simple library app using javascript, HTML, and skeleton CSS framework. 🟢 
You can add books to the table, remove them or change their read status.

### CODE REVIEW

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Invalid
Please use this file [linters.yml](https://github.com/microverseinc/linters-config/blob/master/javascript/.github/workflows/linters.yml) as a linter config.
In your readme add more description about the project.

This CR will be marked invalid, please ask for new CR when you have made the change.

File: main.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  const addBookToLibrary = () => {
+    myLibrary.push({
+      author, title, pages, read,
+    });
+  };
+
+  return {
+    addBookToLibrary, author, title, pages, read, getBooks,
+  };
+};
```
Comment (rindrajosia) [inline main.js]: here's a correct way to write a constructor function:
```
function Book(name, author, pages, read) {
    this.name = name;
    this.author = author;
    this.pages = pages;
    this.read = read;
}
```

File: main.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+    row.innerHTML = `
+        <td>${book.title}</td>
+        <td>${book.author}</td>
+        <td>${book.pages}</td>
+        <td><a href="#" class="change">${book.read}<a></td>
+        <td><a href="#" class="delete">X<a></td>`;
+
+    list.appendChild(row);
+  };
```
Comment (rindrajosia) [inline main.js:21]: It has a function that `loops through myLibrary`

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Changes Required :red_circle:
To Highlight: :angel:

- Good design :+1: 

To improve :factory_worker: [REQUIRED]:

- [x] It has a constructor ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/book_array_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/library)). To learn more[ read Here.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions). (check inline comments below :arrow_double_down:)
- [x] **addBookToLibrary** function ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/book_array_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/library)). It stores the new Book objects into an array. (new Book object is like this `new Book(title, author, pages, read)`)
- [x] It has a function that `loops through the array` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/books_render_loop.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/library)). In your APP, you add directly the new book into the table. (check inline comments below :arrow_double_down:)

Nice to have [OPTIONAL] : 

- [ ]  It has data persistence by saving, retrieving, and updating the library books using the [localstorage API.](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API) ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/localstorage_stretch_requirement.png) from [Odin Project](https://www.theodinproject.com/courses/javascript/lessons/library))

That's it, you can submit another code review request after making these changes :recycle: :innocent:
With all my respect :pray:
Rindra Josia

File: main.js (hunk 2 above)
Comment (addod19) [reply main.js:21]: Kindly address this comment

File: main.js (hunk 1 above)
Comment (addod19) [reply main.js]: done

File: main.js (hunk 2 above)
Comment (MarkoNS1990) [reply main.js:21]: I have that function, it is Book.prototype.displayBooks()

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi @MarkoNS1990, great job so far on this project :100: The constructor function created well :+1: The addBookToLibrary function stores the new Book object into an array as mentioned in the previous review :smile: The is a function that loops through the array and displays each book :slightly_smiling_face: Awesome :+1: 

I hereby approve the project :1st_place_medal: 
Kindly merge it into dev and do not forget to submit the project completion form :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**